README.rst

```diff
@@ -582,19 +639,60 @@ Response Body
 The `statement`_ object.
 
 
+GET /sessions/{sessionId}/statements/{statementId}
+--------------------------------------------------
+
+Returns a specified statement in a session.
+
+Response Body
+^^^^^^^^^^^^^
+
+The `statement`_ object.
+
+
+POST /sessions/{sessionId}/statements/{statementId}/cancel
+----------------------------------------------------------
+
+Cancel the specified statement in this session.
+
+Response Body
+^^^^^^^^^^^^^
+
++------+----------------------------+--------+
+| name | description                | type   |
++======+============================+========+
+| msg  | denote cancelled statement | string |
```

Comment:
In the current code, it will always return "cancelled" as the response of this request. User should query statement state to get the actual state of this statement. 

The code be improved, but I would like to make the doc correctly reflect the current code.